crates/sui/src/upgrade_compatibility.rs

```diff
@@ -348,6 +439,124 @@ impl CompatibilityMode for CliCompatibilityMode {
     }
 }
 
+/// A Compatibility mode used for checking inclusion checks for 'additive' (Subset) or 'dependency only' (Equal) upgrades
+struct CliInclusionCheckMode {
+    errors: Vec<UpgradeCompatibilityModeError>,
+}
+
+impl Default for CliInclusionCheckMode {
+    fn default() -> Self {
+        Self { errors: vec![] }
+    }
+}
+
+impl InclusionCheckMode for CliInclusionCheckMode {
+    type Error = Vec<UpgradeCompatibilityModeError>;
+
+    fn module_name_mismatch(&mut self, _old_address: &Identifier, _new_address: &Identifier) {}
```

Comment:
Is a module name mismatch even possible? I thought we performed this check by first lining up modules by their name. Should this case be removed from the interface?